javascript/javascript3/week3/shoppingCart/shoppingCart.js

```diff
@@ -0,0 +1,96 @@
+class Product {
```

Comment:
Good.
A few comments.
Be careful with your naming. 
add, remove and search Product are well named and do only what that say they do.
getTotal is what? total price? Try to help the reader as much as possible
In renderProducts you could separate the concerns a bit more. For instance create a function that creates the string to be rendered and another that creates the elements
Try to avoid names like p and div.
getUser is doing more than one thing. Also the verb 'get' implies a return value